buildSrc/src/main/java/com/uber/okbuck/generator/BuckFileGenerator.java

```diff
@@ -210,8 +204,8 @@ public static void generate(
 
     libRules.forEach(
         rule -> {
-          if (rule instanceof ResourceRule && rule.name() != null) {
-            deps.add(rule.buckName());
+          if (rule instanceof AndroidModuleRule && rule.name() != null) {
+            deps.add(rule.buckName().replace(":src_", ":res_"));
```

Comment:
should this be moved to exopackage's macro?

Reply:
these `deps` relates to `android_binary` (normal one or binary that correspond to a library instrumentaion apk) rule, not exopackage.
We could do that, but I didn't want to make a breaking change to whoever already override it today (forcing them to modify their defs)